src/styles/main.scss

```diff
@@ -1,3 +1,47 @@
+$cell-size:100px;
+$week-day: "mon" 1, "tue" 2, "wed" 3, "thu" 4, "fri" 5, "sat" 6, "sun" 7;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  margin: auto;
+  flex-wrap: wrap;
+  width: 720px;
+  gap: 1px;
+  border: 1px solid black;
+
+  &__day {
+    display: flex;
+    width: $cell-size;
+    height: $cell-size;
+    background-color: #eee;
+    border: 1px solid black;
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translate(0,-20px);
```

Comment:
Add the transition with a duration of 0.5s to animate the transform property.

Reply:
done!)